pkg/objectstore/objectstore_test.go

```diff
@@ -48,9 +49,11 @@ type ObjectStoreProviderSuite struct {
 const (
 	testBucketName = "kio-store-tests"
 	testRegionS3   = "us-west-2"
+	testRole       = ""
```

Comment:
We need to create a role for Kanister test purpose, and make it settable by environment variables.